docs/sdk/other-apis/android.md

```diff
@@ -133,7 +167,17 @@ AppCenter.setMaxStorageSize(20 * 1024 * 1024L).thenAccept(new AppCenterConsumer<
         // The success parameter is false when the size cannot be honored.
     }
 });
-AppCenter.start("{your secret}", Analytics.class);
+AppCenter.start("{Your App Secret}", Analytics.class);
+```
+```kotlin
+// Use 20MB for storage.
+AppCenter.setMaxStorageSize(20 * 1024 * 1024).thenAccept(object : AppCenterConsumer<Boolean> {
+    override fun accept(success: Boolean) {
+        // The success parameter is false when the size cannot be honored.
+    }
+})
```

Comment:
Android Studio suggests to convert this into a lambda, e.g.
```kotlin
// Use 20MB for storage.
        AppCenter.setMaxStorageSize(20 * 1024 * 1024).thenAccept {
            // The success parameter is false when the size cannot be honored.
        }
```
How about we use the lambda in our sample?

Reply:
👍